MailSim/MailSim/ConfigurationFile.cs

```diff
@@ -13,6 +13,7 @@
 using System.Xml.Serialization;
```

Comment:
This file was accidentally committed in one of its previous versions. I will fix this in a final pull request.